subsys/bluetooth/controller/ll_sw/nrf.cmake

```diff
@@ -100,3 +100,9 @@ zephyr_library_include_directories(
   ll_sw/nordic
   hci/nordic
 )
+
+# This definition needs to be set globally as it is supposed to be used
+# in nrfx_glue.h also when other libraries are built.
+zephyr_compile_definitions(
+  BT_CTLR_USED_RESOURCES_INCLUDE_FILE="${CMAKE_CURRENT_SOURCE_DIR}/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h"
+)
```

Comment:
Not fond of such approach.
I know it's used for example by mbedTLS, but there it's clearly a configuration file, and named `MBEDTLS_CONFIG_FILE`.

In Zephyr we have Kconfig --> `autoconf.h`.

Currently I don't see why the header cannot have an expected name, and then downstream controller implementation can specify the include path that Zephyr should use.

Reply:
I don't mind setting a fixed name for this file. Such approach will be indeed cleaner. I updated the PR accordingly. Please have another look.